### PR TITLE
v2.0

### PR DESCRIPTION
**Pull request by cermakjiri**
# Start from scratch with eslint-config-react-app

The configuration has been updated to only dependency `eslint-config-react-app`, as we discussed last time.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,151 +1,6 @@
...
-
-        // https://eslint.org/docs/rules/curly
-        curly: ['warn', 'all'],
-
-        /**
-         * REASON: https://ackee.slack.com/archives/C07BZ9K32/p1536067640000100
-         */
-        'import/prefer-default-export': 'off',
-    },
+  extends: ["react-app"],
```
Comment (jstorm31) [inline index.js]: Jak moc se tohle bude lišit od dosavadních pravidel, které jsme měli definované?

**Comment by horaklukas** [comment]
@cermakjiri @jstorm31  Hi guys, sorry but I haven't had time to check this request but I would like to ask you for one change: please keep the english language in comments, descriptions, etc. This is an open source repository and we used to communicate in english here, thx 😉

File: index.js (hunk 1 above)
Comment (cermakjiri) [reply index.js]: Hard to say honestly. The previous version had totally different dependencies and default set of rules. Thus, I didn't want to preserve them, instead I'd suggest to start from scratch and fill in custom rules continuously. 

However, we are already using the `react-app` dependency as the __only one__ on every react project, the previous version of `@ackee/eslint-config` was used to only display errors and warnings in editor (not in console).

Comment (jstorm31) [reply index.js]: Okay, good 👍

**Pull request merged**